apps/evm/lib/evm/functions.ex

```diff
@@ -137,6 +137,7 @@ defmodule EVM.Functions do
     end
   end
 
+  # credo:disable-for-next-line
```

Comment:
What was credo complaining about here?

Reply:
`Function is too complex (CC is 13, max is 10).` EIPs introduce too many new operations